jsx/Form.js

```diff
@@ -1085,11 +1106,11 @@ DateElement.propTypes = {
 DateElement.defaultProps = {
   name: '',
   label: '',
-  value: '',
+  value: null,
   id: null,
-  maxYear: '9999',
-  minYear: '1000',
-  dateFormat: 'YMd',
+  maxYear: '9999-12-31',
+  minYear: '1000-01-01',
+  today: true,
```

Comment:
not in **24.1**

_added to improve form functionality and UI_

to include as **CBIG override** with plans to push to **main** in some capacity